sonar-html-plugin/src/main/java/org/sonar/plugins/html/node/Node.java

```diff
@@ -39,7 +37,7 @@ protected Node(NodeType nodeType) {
   }
 
   public String getCode() {
-    return code;
+    return code == null ? "" : code;
   }
```

Comment:
_minor_ I'd rather handle the `null` value in the setter `setCode` :
```java
  public void setCode(String code) {
    this.code = code == null ? "" : code;
  }
```
Else, you could have a null value returned by the `toString` function.

Reply:
Good point for the `toString` method. I'll simply call `getCode` inside it instead, if it makes sense with you

Reply:
Oh, my bad, I didn't think about the possibility of the variable not being initialized 🙄. Thanks for the explanation.